appinventor/components/src/com/google/appinventor/components/runtime/TextBoxBase.java

```diff
@@ -510,6 +574,23 @@ public void RequestFocus() {
 
   // OnFocusChangeListener implementation
 
+  /* Detects for any text changes. Downside is that, all amendments to the text of the TextBox
+  * must be done through the Text() property.*/
```

Comment:
Code style requires the closing `*/` to be on its own line here.

Reply:
Updated.